### ISSUE TITLE
[FaloopIntegration] Spawn reports from all DC when currentDataCenter is unknown

### ISSUE THREAD

**Issue by drejjmit** [SlashNephy/Divination#657]
When ``currentDataCenter == null`` while relogging an alt it prints spawn reports from all worlds. Not previous datacenter or your main datacenter.
In this case i logged out of EU. And then logged in another character on EU. But while the relog was happening i got a report from Oceania, that was waiting in chat when i got in.
Not a huge deal.
But maybe don't print when ``currentDataCenter == null`` or use previously known datacenter as a filter.
```
12:15:34.369 | DBG | [FaloopIntegration] Event "message" = "[{\"type\":\"mob\",\"subType\":\"report\",\"data\":{\"action\":\"spawn\",\"id\":{\"mobId\":\"okina\",\"worldId\":\"sephirot\"},\"data\":{\"zoneId2\":\"the_ruby_sea\",\"zonePoiIds\":[790],\"timestamp\":\"2024-04-30T10:15:34.151Z\",\"window\":1,\"stage\":null}}}]"
12:15:34.369 | DBG | [FaloopIntegration] OnMobReport: currentWorld == null || currentDataCenter == null
```
![image](https://github.com/SlashNephy/Divination/assets/15526476/5eddcaba-20e5-43bf-bc9c-675a010961e0)

**Comment by drejjmit** [comment]
Sorry for all the reports xD 
But i'm under the impression that it's always good to have help with finding uncommon issues ^^

**Comment by SlashNephy** [comment]
This issue was resolved in v6.58.12.
https://github.com/SlashNephy/Divination/releases/tag/6.58.12
